python/lsst/ap/association/dia_object.py

```diff
@@ -83,39 +97,34 @@ def __init__(self, dia_source_catalog, object_source_record=None):
         self._updated = False
 
         if object_source_record is None:
-            self._dia_object_record = afwTable.SourceRecord(
-                make_mimimal_dia_object_schema())
-            self._generate_property_methods()
+            self._dia_object_record = afwTable.SourceTable.makeRecord(
+                afwTable.SourceTable.make(
+                    make_minimal_dia_object_schema()))
+            # For now we copy the first DIASource's object id as our DIAObject
+            # id.
+            self._dia_object_record.set(
+                'id', self._dia_source_catalog[0].getId())
             self.update()
         else:
             self._dia_object_record = object_source_record
-            self._generate_property_methods()
             self._updated = True
 
-    def _generate_property_methods(self):
-        """ Generate methods for accessing values stored in the
+    def get(self, name):
+        """ Return the data stored in column name within the internal
```

Comment:
Current text changed to "Retrieve a specific summary statistic value from this DIAObject."